### DISCUSSION TITLE
Can you add this feature? For better way to code

### DISCUSSION THREAD

**Discussion by coswat** [Acode-Foundation/Acode#621] [Ideas]
Can anyone add a function in acode, when we click a namespace of path it opens that file

**Comment by bajrangCoder** [comment]
Added bro ❤️❤️
Check out `Path Linker` Plugin...

**Reply by morteza-ayashi** [reply]
This plugin just works for file paths not namespaces

**Reply by bajrangCoder** [reply]
Very soon it will added❣️👍

**Reply by morteza-ayashi** [reply]
It would be awesome 👌
